CONTRIBUTING.md

```diff
@@ -77,6 +77,7 @@ git checkout v${ver}
 cd -
 
 sed s/"frequenz-api-common == [0-9]\.[0-9]\.[0-9]"/"frequenz-api-common == ${ver}"/g -i pyproject.toml
+sed 's|https://frequenz-floss.github.io/frequenz-api-common/v[0-9].[0-9]/objects.inv|https://frequenz-floss.github.io/frequenz-api-common/v'${ver_minor}'/objects.inv|' -i mkdocs.yml
```

Comment:
```suggestion
```

This one is not needed because we are actually at the `api-common` repo, this one is special, the templates are for *real* API repos, another reason to name this repo differently.